dev/tests/integration/testsuite/Magento/Eav/Setup/AddOptionToAttributeTest.php

```diff
@@ -0,0 +1,228 @@
+<?php declare(strict_types=1);
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Eav\Setup;
+
+use Magento\Eav\Api\Data\AttributeInterface;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+
+class AddOptionToAttributeTest extends \PHPUnit\Framework\TestCase
```

Comment:
Please add some cases in case that operation throw exception.

Reply:
My modifications do not contain any logic which throws any exceptions.